src/pages/Account/test/Account.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import 'jest-styled-components'
+
+import { App } from '../../../App'
+import urls from '../../../constants/urls'
+import { renderWithRouter } from '../../../utilsTest/render'
+import { configureStore } from '../../../store'
+import { USER } from '../../../utilsTest/mockData'
+
+describe('[pages] Account', () => {
+  describe('when not logged in', () => {
+    it('should redirect to login page', () => {
+      const renderer = renderWithRouter(<App />, urls.myAccount)
+      const HTMLDivElement = renderer.getByText('Sign In')
+      expect(HTMLDivElement).toBeTruthy()
```

Comment:
Very small note: `expect` construction (and asserting it with truthy value) is unnecessary here. It is because getByText throws an error if it don't find matching text and this error will cause test to fail. It can help with readability of tests.
So better test can look like: 
```js
it('should redirect to login page', () => {
  const renderer = renderWithRouter(<App />, urls.myAccount)
  renderer.getByText('Sign In')
})
```

Reply:
FIxed